src/components/PopularMovie.tsx

```diff
@@ -44,7 +45,16 @@ const PopularMovie: React.FC<PopularMovieProps> = ({ movie }) => {
           className={`movie-description ${
             window.innerWidth < 767 ? 'text-overlow' : ''
           }`}>
-          {overview}
+          {overview.length >= 250 ? (
+            <p>
+              {overview.slice(0, 250)}
+              <Link className="movie-read-more" to={`/movie-details/:${id} `}>
+                <span>...Read more</span>
+              </Link>
+            </p>
+          ) : (
+            overview
+          )}
```

Comment:
I belive @solomiia-sahala did such changes in separate component, please syncup and align changes

Reply:
Here I am using a simpler version of Solomia's component because I just need to show a Read more btn when a length of description is more than 250 symbols and a pressing to a Read more btn leads to Movie Detail page instead of just spreading a description and change Read more to Show Less